packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -467,6 +491,12 @@ const double _kTrackInnerEnd = _kTrackWidth - _kTrackInnerStart;
 const double _kTrackInnerLength = _kTrackInnerEnd - _kTrackInnerStart;
 const double _kSwitchWidth = 59.0;
 const double _kSwitchHeight = 39.0;
+const double _kOnLabelWidth = 2.0 / 2.0;
+const double _kOnLabelHeight = 20.0 / 2.0;
+const double _kOnLabelPaddingHorizontal = 22.0 / 2.0;
+const double _kOffLabelWidth = 2.0 / 2.0;
+const double _kOffLabelPaddingHorizontal = 24.0 / 2.0;
+const double _kOffLabelRadius = 10.0 / 2.0;
```

Comment:
Is there a reason why these are written as a function instead of just the value? As in, `1.0` instead of `2.0 / 2.0`.

Reply:
Yes, this is because `22.0`, `2.0`, `24.0` and `10.0` are the actual size in the inspector,
see https://github.com/flutter/flutter/issues/4830#issuecomment-528495360

I'm happy to change these values to `11.0`, `1.0`, etc. though, should I do that?

Reply:
I think so. It will be a bit more clear. Adding a comment on where those values came from could be helpful as well.